impress-org/givewp
# Custom amount level is correctly set after a payment error

Revisits #5912, after changes included by #5913, in response to https://github.com/impress-org/givewp/pull/5913#issuecomment-897352881 and https://github.com/impress-org/givewp/issues/5912#issuecomment-898553249.

## Description



We can not figure out custom amount after redirection. For this reason we add `custom-amount` to redirect url. This helps us to reset `custom` level id and donor custom amount after redirection. `autoSetMultiLevel ` does good job and set donation amount correctly. For this reason instead of manual set focus on amount field and exit early from function, we only setup correct donation amount when level id set to `custom` and rest will be handle by function.

## Testing Instructions



- Set a custom donation amount
- Submit payment with payment error
- Re-submit with successful card - no errors
- Form should submit correctly with the custom amount.

## Pre-review Checklist
